### ISSUE TITLE
Some question about downstream task

### ISSUE THREAD

**Issue by mn7ark** [s3prl/s3prl#128]
Hi, thanks for the project, it's so helpful.
There are two question that I want to ask.
1. Does the project provide some metric that take less training time? e.g. ABX, or some metric that is used in zerospeech2021(I note that this is one of the downstream task, but it seems it hasn't been completed yet.)
2. When I perform evaluation on downstream task, I find that dev/test score may be quite different. I just want to make sure whether all result in  SUPERB used dev set only for choosing best step, and report result on test set(I didn't find description about this on the paper).

**Comment by leo19941227** [comment]
Hi,

Thank you for the interest!

1. Unfortunately we haven't supported that yet, and we will put that into the schedule. Thanks for the suggestion!
2. Yes! We only save the checkpoints basing on the development metrics. May I ask in which tasks you encountered this issue? I can double check that.

**Comment by mn7ark** [comment]
Thanks for the quick reply!
I ran downstearm IC task on fluent speech command dataset, and found that the dev/test accuracy may be different for about 10%(dev is lower).

**Comment by andi611** [comment]
> I ran downstearm IC task on fluent speech command dataset, and found that the dev/test accuracy may be different for about 10%(dev is lower).

Hi, the difference in performance is most likely caused by the learning rate.
In the SUPERB paper, we have tuned the _lr_ for each upstream.
You may not get the best accuracy with the default _lr_ (depending on the upstream you are using, different upstream prefer different _lr_).

Andy
